### PR TITLE
[source-mysql-v2] : Ensure parity of cdc metadata fields with old connector

### PR DESCRIPTION
**Pull request by akashkulk**
See v1 definitions : https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/MySqlSource.java#L279

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties
Comment (rodireich) [inline gradle/wrapper/gradle-wrapper.properties:1]: Pipeline seems to be holding.
Previously I think it used to depend on 8.6

File: airbyte-integrations/connectors/source-mysql-v2/src/main/kotlin/io/airbyte/integrations/source/mysql/MysqlJdbcStreamFactory.kt [hunk 1]
```diff
@@ -45,8 +44,8 @@ class MysqlJdbcStreamFactory(val base: JdbcAirbyteStreamFactory) : AirbyteStream
         override val type: FieldType,
     ) : MetaField {
         CDC_CURSOR(CdcIntegerMetaFieldType),
-        CDC_UPDATED_AT(CdcOffsetDateTimeMetaFieldType),
-        CDC_DELETED_AT(CdcOffsetDateTimeMetaFieldType),
+        CDC_UPDATED_AT(CdcStringMetaFieldType),
```
Comment (rodireich) [inline airbyte-integrations/connectors/source-mysql-v2/src/main/kotlin/io/airbyte/integrations/source/mysql/MysqlJdbcStreamFactory.kt:47]: For my understanding: this is going to put json values in quotes?

**Review by rodireich — APPROVED** [review]

File: airbyte-integrations/connectors/source-mysql-v2/src/main/kotlin/io/airbyte/integrations/source/mysql/MysqlJdbcStreamFactory.kt (hunk 1 above)
Comment (akashkulk) [reply airbyte-integrations/connectors/source-mysql-v2/src/main/kotlin/io/airbyte/integrations/source/mysql/MysqlJdbcStreamFactory.kt:47]: yes I believe so

File: gradle/wrapper/gradle-wrapper.properties
Comment (akashkulk) [reply gradle/wrapper/gradle-wrapper.properties:1]: I'll revert

**Pull request merged**